### PR TITLE
Add post mortem

### PR DESCRIPTION
**Pull request by yngvark**
## Description

Add post mortem for issue connected to EKS upgrade.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have updated the release notes (for the next release).

### CODE REVIEW

File: docs/post-mortems/0005-app-stopped-working-after-eks-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+<!--
+Describe how the problem was fixed.
+
+Example:
+- Manually scaled up number of backend instances letting regular traffic stay unaffected
+- Created a fix for the database schema
+-->
+
+We updated to the correct application configuration:
+* removed `dnsPolicy: true` from all three application's Deployment.
```
Comment (deifyed) [inline docs/post-mortems/0005-app-stopped-working-after-eks-upgrade.md:83]:
```suggestion
* removed `dnsPolicy: default` from all three application's Deployment.
```

File: docs/post-mortems/0005-app-stopped-working-after-eks-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+Describe what went well trying to handle the event.
+
+Example:
+- Alerting mechanisms worked brilliantly when errors started comming in
+- Deploying the database update was fast
+-->
+
+* We knew by memory what a correct application configuration was. This would also be possible to aquire by running `okctl apply application` with an existing Application manifest, or generating a new on with `okctl scaffold application`. 
+* We ran the EKS upgrade in the development environment before the production environment
+* In the AWS account where we did the upgrade, we only has "Basic" support level, which probably would have been to slow to use. Luckily, someone on Slack was available to tell how to upgrade it - though we ended up fixing it before this became necessary.
```
Comment (deifyed) [inline docs/post-mortems/0005-app-stopped-working-after-eks-upgrade.md:98]: If you log into the AWS account with the read only role, the support level will be shown as Basic. However, in some accounts (not quite sure if all or just a select few), if you use the admin role the support level will be shown as Business

**Review by deifyed — APPROVED** [review]
The index of the file collides with a different post mortem.

Feel free to merge when comments are resolved. Interesting read!

File: docs/post-mortems/0005-app-stopped-working-after-eks-upgrade.md (hunk 2 above)
Comment (yngvark) [reply docs/post-mortems/0005-app-stopped-working-after-eks-upgrade.md:98]: Thanks for the tip. (I was using admin role, but didn't know about this.)

**Pull request merged**